examples/contracts-and-usage-policy/templates/TargetTemplates/AssetTemplateShape.ttl

```diff
@@ -14,7 +14,7 @@ shapes:AssetTemplateShape
 	a sh:NodeShape ;
 	sh:or (
 		[
-			sh:nodeKind sh:IRI ; # it's an RDF URI
+			sh:datatype xsd:anyURI ; # it's an RDF URI
```

Comment:
I am not sure if this change is correct. `sh:IRI` is actually semantically correct IMHO. CodeGen should support that, or not @changqin26 ?